src/controller/CHIPDeviceController.cpp

```diff
@@ -948,6 +955,7 @@ void DeviceCommissioner::OnWiFiPAFSubscribeError(void * appState, CHIP_ERROR err
     {
         ChipLogError(Controller, "WiFi-PAF: Subscription Error, id = %lu, err = %" CHIP_ERROR_FORMAT, device->GetDeviceId(),
                      err.Format());
+        self->CloseWiFiPAFConnection(device->GetDeviceId());
```

Comment:
Unfortunately, the id of a device is not necessarily stable.  See `CommissioneeDeviceProxy::SetPeerId`: it can change after AddNOC.

I suppose it's safe enough to do that here because the device id can't change before PASE is established.  But having CloseWiFiPAFConnection() take the device id is inviting bugs in the future.

It would be much better if the PeerAddress knew what thing we are trying to talk to here and we used the PeerAddress as the thing we pass into CloseConnection bits....  That's what NFC is planning to implement. 

Looking at SetUpCodePairer, it looks like a NodeId may be input to PAF when doing discovery, so the PeerAddress could just include that (stable, since it's in the PeerAddress and not going to change) ID, right?

Reply:
I will change to use PeerAddress to get the id